.circleci/config.yml

```diff
@@ -2,17 +2,51 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/golang:1.10
+      - image: circleci/golang:1.12.5
```

Comment:
Does Circle let us carrot this?

Reply:
I don't believe so, I can't find anything online indicating this is possible

Reply:
We are pegging the version the sdk is using (1.12.5)

Reply:
Actually there is a latest tag for this image, but I'm not sure that is what we would want to do:
https://circleci.com/docs/2.0/circleci-images/#go-golang

Reply:
`1.12.6` is available but looking at the sdk they are sticking to 1.12.5 https://github.com/cosmos/cosmos-sdk/blob/master/.circleci/config.yml